src/cmap/establish/handshake/test.rs

```diff
@@ -0,0 +1,61 @@
+use bson::doc;
```

Comment:
these tests are good, but I think we should also have some integration tests w/ command monitoring so that we get "end to end" coverage so to speak.

Reply:
I originally was going to implement them with that, but since command monitoring only happens in `execute_operation`, it doesn't happen during the handshake (which uses `send_command` directly).